x-pack/plugins/security_solution/public/resolver/view/clickthrough.test.tsx

```diff
@@ -242,6 +242,7 @@ describe('Resolver, when analyzing a tree that has two related events for the or
         );
         if (button) {
           button.simulate('click');
+          button.simulate('click'); // The first click opened the menu, this second click closes it
```

Comment:
When I updated the extended jest file to retain a 'success' state once one is reached, this test began failing.